src/io/flutter/run/FlutterRunningState.java

```diff
@@ -106,4 +117,25 @@ public static VirtualFile flutterProjectDir(@Nullable String projectPath) throws
   public int getObservatoryPort() {
     return myObservatoryPort;
   }
+
+  private class OpenIOSSimulatorLink implements HyperlinkInfo {
+    @Override
+    public void navigate(Project project) {
+      final GeneralCommandLine cmd = new GeneralCommandLine().withExePath("open").withParameters("-a", "Simulator.app");
+      try {
+        final OSProcessHandler handler = new OSProcessHandler(cmd);
+        handler.addProcessListener(new ProcessAdapter() {
+          @Override
+          public void processTerminated(final ProcessEvent event) {
+            //TODO: schedule a re-run or notify user to try again.
```

Comment:
Perhaps just a temporary toast telling the user that the simulator opened? Does IntelliJ have ~3 second auto-closing toasts?